meson_options.txt

```diff
@@ -208,6 +208,8 @@ option('status-unit-format-default', type : 'combo',
        description : 'use unit name or description in messages by default')
 option('time-epoch', type : 'integer', value : '-1',
        description : 'time epoch for time clients')
+option('clock-valid-range-max', type : 'integer', value : '473364000', # 15 years
+       description : 'maximum value in seconds for the difference between rtc and epoch, exceeding which is considered an rtc error')
```

Comment:
we generally unify on usec time times, please use that here too. and please also include USEC in the name of the setting to indicate the unit: CLOCK_VALID_RANGE_USEC_MAX or so